CONTRIBUTING.md

```diff
@@ -196,6 +196,22 @@ In this example this can be done by invoking following command from the reposito
 ```bash
 ./gradlew test -Pcoordinates=org.example:library:0.0.1
 ```
+ 
+### Providing the tests that use docker
+ 
+If your tests use docker (either with explicit docker process invocation or through some library method call), all images 
+have to be declared in `required-docker-images.txt` file. This file must be placed under `/tests/src/<groupId>/<artifactId>/<versionId>`.
+
+Only docker images that are listed [here](https://github.com/oracle/graalvm-reachability-metadata/blob/master/tests/tck-build-logic/src/main/resources/AllowedDockerImages.txt)
+can be executed. If you want to extend this list, please create separate pull request to do that.
```

Comment:
Perhaps a comment that we use Grype on all images under this file, and if contributors modify this file they should post the Grype output for their newly added image